WittyWaithira/MyStoreManager
# User Interface

What does this PR do?
It Creates a Login page for both the Store attendant and the Store owner.

Description of Task to be completed?
The user should be able to login using a valid Email Address and Password. If the Login credentials are false there is an error to the system.

How should this be tested?
Clone the repository to you local machine then access the page using your local URL to your a browser. Try to log in.

Background context you want to provide?
It is a requirement with the project guidelines that the Login credentials should be validated.

What are the relevant pivotal tracker stories?
#161035959
